snapcraft/cli/store.py

```diff
@@ -360,19 +362,39 @@ def close(snap_name, channels):
 
 
 @storecli.command()
+@click.option(
+    "--experimental-progressive-delivery",
+    is_flag=True,
+    help="*EXPERIMENTAL* Enables 'progressive releases'.",
+    envvar="SNAPCRAFT_EXPERIMENTAL_PROGRESSIVE_DELIVERY",
+)
 @click.option(
     "--arch", metavar="<arch>", help="The snap architecture to get the status for"
 )
 @click.argument("snap-name", metavar="<snap-name>")
-def status(snap_name, arch):
+def status(snap_name, arch, experimental_progressive_delivery):
     """Get the status on the store for <snap-name>.
 
     \b
     Examples:
         snapcraft status my-snap
-        snapcraft status my-snap --arch armhf
     """
-    snapcraft.status(snap_name, arch)
+    if experimental_progressive_delivery:
+        os.environ["SNAPCRAFT_EXPERIMENTAL_PROGRESSIVE_DELIVERY"] = "Y"
```

Comment:
This is already handled by apply_host_provider_flags(), but I have used strtobool, so they could be made consistent to 0|1 for boolean flags for checking throughout code.

Reply:
We discussed this and agreed that this only applies to build commands (this is not one) and that one solution would be to implement a click.Option that does this for us.